src/python/director/segmentation.py

```diff
@@ -199,12 +202,13 @@ def onRemoveFromObjectModel(self):
         self.timer.stop()
 
     def update(self):
-
         utime = self.imageManager.queue.getCurrentImageTime(self.cameraName)
         if utime == self.lastUtime:
+            if self.getProperty('Remove Stale Data') and ((time.time()-self.lastDataReceivedTime) > self.getProperty('Stale Data Timeout')):
+                self.setPolyData(vtk.vtkPolyData())
```

Comment:
maybe also add a check for self.polyData.GetNumberOfPoints() > 0 so that it does not continuously reset the polyData?  Not sure there's really anything wrong with it as written tho.